mongo-rs.env

```diff
@@ -0,0 +1,7 @@
+# Env variables to customise the recipe.
+STACK_NAME="mongo"
+MONGO_SERVICE_URI="mongo-rs_mongo"
```

Comment:
@chicco785 I already approved but forgot to mention, `MONGO_SERVICE_URI` should actually be `STACK_NAME` + "mongo".

Reply:
it should be fixed